### PR TITLE
Day11 Note

### PR DESCRIPTION
**Pull request by haingo94**
#### Description (Required)
- [x] create and handle database
- [x] create screen note, new note and note detail
- [x] event new note, edit and delete note

#### Reference (Optional)

### CODE REVIEW

File: app/src/main/java/vn/asiantech/internship/databases/DatabaseHelper.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    public static final String COLUMN_NOTE_IMAGE_URI = "Image_Uri";
+
+    public DatabaseHelper(Context context) {
+        super(context, DATABASE_NAME, null, DATABASE_VERSION);
+    }
+
+    @Override
+    public void onCreate(SQLiteDatabase db) {
+        String sql = "create table " + TABLE_NOTE + "(" + COLUMN_NOTE_ID + " Integer primary key, "
+                + COLUMN_NOTE_DAY_OF_WEEK + " text, " + COLUMN_NOTE_DAY_OF_MONTH + " text, "
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/internship/databases/DatabaseHelper.java]: WARNING: [AvoidDuplicateLiterals] The String literal " text, " appears 5 times in this file; the first occurrence is on line 31
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/strings.html#AvoidDuplicateLiterals

File: app/src/main/java/vn/asiantech/internship/ui/note/fragment/NewNoteFragment.java [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+                break;
+        }
+    }
+
+    private void intentGallery() {
+        Intent intent = new Intent();
+        intent.setType("image/*");
+        intent.setAction(Intent.ACTION_GET_CONTENT);
+        startActivityForResult(intent, REQUEST_CODE_GALLERY);
+    }
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/internship/ui/note/fragment/NewNoteFragment.java]: ERROR: [PMD-CPD] 29 lines duplicated.
app/src/main/java/vn/asiantech/internship/ui/note/fragment/NoteDetailFragment.java

File: app/src/main/java/vn/asiantech/internship/ui/note/fragment/NoteDetailFragment.java [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+    }
+
+    private void setNote(Note note) {
+        mEdtEditTitle.setText(note.getTitle());
+        mEdtEditContent.setText(note.getContent());
+        if (note.getImage() != null) {
+            mImgNote.setVisibility(View.VISIBLE);
+            mImgNote.setImageURI(Uri.parse(note.getImage()));
+        }
+    }
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/internship/ui/note/fragment/NoteDetailFragment.java]: ERROR: [PMD-CPD] 29 lines duplicated.
app/src/main/java/vn/asiantech/internship/ui/note/fragment/NewNoteFragment.java

File: app/src/main/java/vn/asiantech/internship/databases/DatabaseHelper.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    public static final String COLUMN_NOTE_IMAGE_URI = "Image_Uri";
+
+    public DatabaseHelper(Context context) {
+        super(context, DATABASE_NAME, null, DATABASE_VERSION);
+    }
+
+    @Override
+    public void onCreate(SQLiteDatabase db) {
+        String sql = "create table " + TABLE_NOTE + "(" + COLUMN_NOTE_ID + " Integer primary key autoincrement, "
+                + COLUMN_NOTE_DAY_OF_WEEK + " text, " + COLUMN_NOTE_DAY_OF_MONTH + " text, "
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/internship/databases/DatabaseHelper.java]: WARNING: [AvoidDuplicateLiterals] The String literal " text, " appears 5 times in this file; the first occurrence is on line 31
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/strings.html#AvoidDuplicateLiterals

**Comment by at-tamnguyen2** [comment]
@haingo94 bài em khá ok rồi, em fix conflicts rồi anh Tiến merge nhé

**Comment by at-tienhoang** [comment]
LGTM! thank you
LGTM

**Pull request merged**